sql/moz-fx-data-shared-prod/subscription_platform_derived/logical_subscriptions_history_v1/metadata.yaml

```diff
@@ -25,13 +25,10 @@ scheduling:
   - ['moz-fx-data-shared-prod', 'subscription_platform_derived', 'recent_subplat_attribution_impressions_v1']
   - ['moz-fx-data-shared-prod', 'subscription_platform_derived', 'stripe_logical_subscriptions_history_v1']
 bigquery:
-  time_partitioning:
-    type: day
-    field: valid_to
-    require_partition_filter: false
-    expiration_days: null
+  time_partitioning: null
```

Comment:
not sure if this will cause problems with deploys as the underlying table is partitioned

Reply:
> I'm planning to manually recreate the tables using `CREATE OR REPLACE TABLE ... AS ...`.

Unfortunately it turns out BigQuery doesn't allow that:
> _Cannot replace a table with a different partitioning spec. Instead, DROP the table, and then recreate it._

So I guess I'm dropping and replacing the tables.